module/canvas/hit-box.mjs

```diff
@@ -0,0 +1,76 @@
+/**
+ * A canvas layer which renders Crucible token hitboxes using the CrucibleHitBoxShader batch plugin.
+ * Displayed above the Grid layer and below the Tokens layer (zIndex 90).
+ */
+export default class CrucibleHitboxLayer extends foundry.canvas.layers.CanvasLayer {
+  constructor() {
+    super();
+    this.eventMode = "none";
+  }
+
+  /** @override */
+  static get layerOptions() {
+    return {name: "crucibleHitboxes", zIndex: 90};
+  }
+
+  /* ---------------------------------------- */
+
+  /**
+   * Collect the tokens whose hitboxes should be drawn.
+   * @returns {Token[]|null} Visible tokens if at least one is controlled, otherwise null
+   */
+  _collect() {
+    const anyControlled = canvas.tokens.controlled.length > 0;
+    if ( !anyControlled ) return null;
+    return canvas.tokens.placeables.filter(t => t?.visible);
+  }
+
+  /* ---------------------------------------- */
+
+  /** @override */
+  _draw() {}
+
+  /* ---------------------------------------- */
+
+  /** @override */
+  _tearDown() {}
+
+  /* ---------------------------------------- */
+
+  /** @override */
+  render(renderer) {
+    if ( !this.worldVisible || this.worldAlpha <= 0 ) return;
```

Comment:
Maybe some protection against being in a scene that is not `canvas.scene.useMicrogrid === true`?